.github/workflows/test_codecov.yml

```diff
@@ -0,0 +1,41 @@
+name: CI test and coverage
+
+on:
+  pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    name: Run tests for ${{ matrix.python-version }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+    - name: Install dependencies and package
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+        pip install -e .
```

Comment:
Do we really need 4 lines to install?

Reply:
... Maybe? One for upgrading, one for dependencies, one for development tools (eg pytest, flake8), one for the package itself — can we wrap this into one?
I am not that fluent in pip commands (this is the result of copy pasting) and this works so i'm happy to leave it, but also very open to suggestions.